### PR TITLE
Add tmux cheats

### PR DESCRIPTION
**Pull request by Baycken**
This is to address #44
* Add explanation for some tmux commands and shortcuts
* Add the ability to copy from tmux copy buffer to the system clipboard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,164 @@ rosconnect [host_name]
...
+- Start new session with name: `tmux new -s <name>`
+- List all sessions:`tmux ls`
+- Reattach most recent disattached session: `tmux a`
+- Reattach a specific session: `tmux a -t <name>`, `<name>` can be found by
+`tmux ls` and can be either string or integer.
+- Kill specific session: `tmux kill-session -t <name>`
+- kill all sessions: `tmux kill-session -a`
+
+### Key-bindings
+The key-bining prefix is changed from `ctrl-b` to `ctrl-a`.
```
Comment (anassinator) [inline README.md]: I think the description and this is the only important part. we can link to a cheatsheet online for the rest.

Comment (Baycken) [inline README.md]: I think it will be nice to have it, we can move it to wiki later if wiki is what we want to do

File: setup/config/tmux.conf [hunk 2]
```diff
@@ -11,6 +11,13 @@ bind - split-window -v
 unbind '"'
```
Comment (foochuanwei) [inline setup/config/tmux.conf]: Do you think we should remove the 'unbinds'?
Having the choice to either do things the default way or the easier way minimizes confusion imho.

Comment (anassinator) [inline setup/config/tmux.conf]: Makes sense, I think that should be in a separate PR though
